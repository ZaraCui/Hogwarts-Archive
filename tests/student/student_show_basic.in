ADD STUDENT Hermione Granger
STUDENT 100000
EXIT

