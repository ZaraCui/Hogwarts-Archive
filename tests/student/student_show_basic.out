user: Success.

user: 100000: Hermione Granger

user: Ending Archive process.
